src/components/catalog-filter/catalog-filter.controller.ts

```diff
@@ -105,10 +105,12 @@ export class CatalogFilterController implements angular.IController {
   };
 
   private highlightKeywordFilter = (keyword: string) => {
-      let filterTags: any = document.querySelectorAll('.active-filter.label.label-info.single-label');
+      let filterTags: any = document.querySelectorAll('pf-filter-panel-results .label-info');
 
       let existingKeyword: any = _.find(filterTags, (tag: any) => {
-        return tag.innerText.trim() === ("Keyword: " + keyword.toLowerCase());
+        let filterTagText: string = tag.innerText.trim();
+        return _.includes(filterTagText, "Keyword:") &&
+               _.includes(filterTagText, keyword.toLowerCase());
```

Comment:
If `filterTagText` is a string, won't this match substrings? For instance, "Keyword:foobar" includes "foo"

Reply:
Ok, changed to be more restrictive.

Reply:
Hi @spadgett, think this is ready to go :-)